module2/exercise_05_text_file/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "module2_exercise_04",
+  "version": "1.0.0",
+  "description": "This is the data selfie app by Joey Lee.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "example",
+    "data",
+    "selfie"
+  ],
+  "author": "Daniel Shiffman",
```

Comment:
You should feel free to credit yourself in the exercise solutions that you are providing!